packages/css/src/plugins/atomicify-rules.ts

```diff
@@ -76,20 +77,28 @@ const replaceNestingSelector = (selector: string, parentClassName: string) => {
  * @param node
  */
 const buildAtomicSelector = (node: Declaration, opts: AtomicifyOpts) => {
+  const { classNameCompressionMap } = opts;
   const selectors: string[] = [];
 
   (opts.selectors || ['']).forEach((selector) => {
     const normalizedSelector = normalizeSelector(selector);
-    const className = atomicClassName(node, {
+    const fullClassName = atomicClassName(node, {
       ...opts,
       selectors: [normalizedSelector],
     });
-    const replacedSelector = replaceNestingSelector(normalizedSelector, className);
 
-    selectors.push(replacedSelector);
+    const compressedClassName =
+      classNameCompressionMap && classNameCompressionMap[fullClassName.slice(1)];
+
+    if (compressedClassName) {
```

Comment:
If compressed class name is available, we are replacing full class name with the compressed in the sheet.
So
```
.aaaabbbb { font-size: 10px }
```
becomes
```
.a { font-size: 10px }
```
It will reduce the size of stylesheet but will break if we don't replace every occurrence of `_aaaabbbb`, which I should have done in this PR. 🤞